### PR TITLE
ci: 9210 actions

### PR DESCRIPTION
**Pull request by Mikemosca**
### Summary

Add github actions support

### CODE REVIEW

**Comment by repjarms** [comment]
@Mikemosca this looks good. I left one comment about removing the `--dry-run` flag from the `semantic-release` command. Also, I noticed that we are updating the `README` to note that the travis build is deprecated, but I'm wondering if we need to delete `.travis.yml` in this PR so that we don't run double CI once it's merged.

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        npm install -g @semantic-release/exec
+        npm install -g @semantic-release/git
+        npm install -g @semantic-release/github
+        npm install -g @semantic-release/commit-analyzer
+    - name: Publish to Git Releases and Tags
+      if: ${{ github.event.workflow_run.conclusion == 'success' }}
+      env:
+        GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      run: npx semantic-release --dry-run
```
[the remark this answers is not in the document]
Comment (Mikemosca) [reply .github/workflows/deploy.yml:48]: I was going to remove the dry-run on the ne3xt PR when I fix the C&C deprecated message.
For this round I want it there to just make sure everything holds together.

**Comment by Mikemosca** [comment]
> @Mikemosca this looks good. I left one comment about removing the `--dry-run` flag from the `semantic-release` command. Also, I noticed that we are updating the `README` to note that the travis build is deprecated, but I'm wondering if we need to delete `.travis.yml` in this PR so that we don't run double CI once it's merged.

Travis has already been disabled in the settings and will not run.  we can leave that file for now as it does nothing.

**Review by repjarms — APPROVED** [review]
LGTM - let's make sure we squash this because there are a lot of commits here

**Comment by CYBERSECURITY1101** [comment]
.

**Pull request merged**